cmd/agentctl/commands/generate.go

```diff
@@ -85,7 +95,7 @@ func runGenerate(cli *AgentCli, opts GenerateOptions) {
 		}
 		out, err = m.MarshalToString(modelInstance)
```

Comment:
ignoring "--oneline" flag. Could be fixed by setting `m.Indent` to an empty string